controllers/saved-pins-controller.js

```diff
@@ -3,3 +3,26 @@ const SavedPins = require('../models/saved-pins');
 /**
  * TODO - export functions to be used for completing requests in ../routes/search.js
  */
+
+exports.getSavedPins = (appReq, appRes) => {
+  SavedPins.find().then((savedPins) => {
+    appRes.send({ savedPins });
+  }, (e) => {
+    appRes.status(400).send(e);
+  });
+};
+
+exports.postSavedPins = (appReq, appRes) => {
```

Comment:
Suggest adding docstring for this function so its clear to the frontend when they're ready to consume this what we expect to receive in the body of the request along with param definitions. This is pretty clear to me reading the code but we will have people from all levels working on this codebase so it would help to setup a standard of cleanly documenting our api functions. The additional benefit is that it works as quick summary of the intent for the function which makes for easy code review and refactoring if needed in the future.